tls/s2n_kem.c

```diff
@@ -46,6 +46,18 @@ const struct s2n_kem s2n_bike1_l1_r2 = {
         .decapsulate = &BIKE1_L1_R2_crypto_kem_dec,
 };
 
+const struct s2n_kem s2n_bike1_l1_r3 = {
+        .name = "BIKE1r3-Level1",
```

Comment:
Is this the name Shay suggested? Because I'd assume we want to go with BIKE_l1_r3, without "1" after BIKE, since that's the official name of the round 3 protocol, simply BIKE (round 2 submission had three versions of the protocol BIKE-1/2/3, but round 3 submission has a single version named BIKE).

Reply:
I was just using the same naming convention that we used for Round 1 and Round 2: https://github.com/aws/s2n-tls/blob/1fb54f88b5e4ce619ef1ee52e19b7de552ba999b/tls/s2n_kem.c#L25-L47

I can rename to `BIKEr3-Level1`. The only place I know of where this is used is when logging the TLS connection settings after establishment. This string is logged so that we can monitor which KEM's are being used.

Reply:
yes, it made sense for Rounds 1 and 2 because the imported version was called BIKE-1. But Round 3 protocol is called BIKE so the struct name and the macros should be renamed to delete "1" after BIKE.